vtds_cluster_kvm/private/scripts/deploy_cluster_to_blade.py

```diff
@@ -32,23 +32,13 @@
 
```

Comment:
REVIEWERS: this file is the base cluster deploy script. It has been mined for common functions which were moved to `cluster_common.py` and also for the code used in the Node Builder and Disk Builder classes. It is one of the python libraries shipped to the blade during the `prepare` phase.